CONTRIBUTING.md

```diff
@@ -71,18 +88,33 @@ Remember to always work in a branch of your local copy, as you might otherwise h
 
 Please also see [GitHub workflow](https://github.com/open-telemetry/community/blob/main/CONTRIBUTING.md#github-workflow) section of general project contributing guide.
 
-### Running the tests
+## Development
+
+### Tools used
+
+- [NPM](https://npmjs.com)
```

Comment:
Everything below is automatically installed, right? Might be good to mention what versions of npm/node are supported/recommended, or possibly have something like a nvmrc file or similar?

Reply:
Good callout - I cheated a little to avoid this getting out-of-date, and suggested links to refer to for dev dependencies and runtimes.